src/Widgets/WindowClone.vala

```diff
@@ -448,6 +380,13 @@ public class Gala.WindowClone : Clutter.Actor {
 
         var window_title_alloc = InternalUtils.actor_box_from_rect (window_title_x, window_title_y, window_title_width, window_title_height);
         window_title.allocate (window_title_alloc);
+
+        var window_icon_size = InternalUtils.scale_to_int (WINDOW_ICON_SIZE, monitor_scale);
+        var window_icon_x = (box.get_width () - window_icon_size) / 2;
+        var window_icon_y = box.get_height () - (window_icon_size * 0.75f);
+
+        var window_icon_alloc = InternalUtils.actor_box_from_rect (window_icon_x, window_icon_y, window_icon_size, window_icon_size);
+        window_icon.allocate (window_icon_alloc);
```

Comment:
Is there a reason to calculate `window_icon_size` every time we allocate the clone? Can we use `window_icon.width` and `window_icon.height` instead?

Reply:
I think the reason was that the scale factor might change e.g. if the window was on another monitor but is shown in a spread on the primary. But I guess since we only change size here it would look bad either way so let's just use the preferred size instead (we should maybe deal with the scale change somewhere else)